ui/views.py

```diff
@@ -57,7 +57,8 @@ def get(self, request, *args, **kwargs):
             "name": name,
             "username": username,
             "host": webpack_dev_server_host(request),
-            "edx_base_url": settings.EDXORG_BASE_URL
+            "edx_base_url": settings.EDXORG_BASE_URL,
+            "search_url": "http://localhost:9200",
```

Comment:
so this is still going to fail for us inferior mac users. trying to think of how we should address this...